openfga/v1/errors_ignore.proto

```diff
@@ -71,6 +71,14 @@ enum ErrorCode {
   unsupported_schema_version = 2057;
 }
 
+enum ThrottledErrorCode {
```

Comment:
This top-level enum shouldn't be so specific, otherwise we'll have error explosion in our API. These top-level enums should map to the categorical code, and the sub-codes therein specific to the specific error.

In this case, we should have `enum ResourceExhaustedErrorCode` with subcode `throttled_request`